docs/notebooks/geolift1.ipynb

```diff
@@ -0,0 +1,614 @@
+{
```

Comment:
<p>company want<strong>s</strong> you to</p><p>uplify -&gt; uplift</p><p> </p><p>Love the narrative and the stakes in this intro!</p><p> </p><p>It turns out that <code>CausalPy</code> does exactly this. -&gt; You find a library called <code>CausalPy</code> that supports exactly that use case and are delighted.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>